src/main/java/com/example/turboaz/services/SubscriptionService.java

```diff
@@ -0,0 +1,46 @@
+package com.example.turboaz.services;
+
+import com.example.turboaz.dtos.SubscriptionDto;
+import com.example.turboaz.dtos.SubscriptionListDto;
+
+import java.util.List;
+
+/**
+ * Subscription service.CRUD subscriptions
+ */
+public interface SubscriptionService {
+    /**
+     * @param subscriptionDto
+     * @param userId
+     * @return SubscriptionDto
+     */
+    SubscriptionDto createSubscription(Long userId, SubscriptionDto subscriptionDto);
+
+    /**
+     * @param subscriptionDto
+     * @param userId
+     * @param subscriptionId
+     * @return SubscriptionDto
+     */
+    SubscriptionDto updateSubscription(Long userId, Long subscriptionId, SubscriptionDto subscriptionDto);
+
```

Comment:
In my opinion, we also should have updateDateOfSubscription, because if user would like to extend his subscription, we need only update its date

Reply:
If the user wants to renew the subscription time, he can use this method